src/Checkbox/HoverMonitor.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+const HoverMonitor = () => {
+  let hoverEnabled = true;
+
+  function enableHover() {
+    if (!hoverEnabled) {
+      hoverEnabled = true;
+    }
+  }
+
+  function disableHover() {
+    if (hoverEnabled) {
+      hoverEnabled = false;
+    }
+  }
+
+  if (typeof document !== 'undefined') {
+    document.addEventListener('touchstart', disableHover, true);
```

Comment:
So this disables your abstraction of `hover` for the whole document, thanks to your last argument being `true`, right?
I think it is quite clever. 
My only critic would be that we might need to clean up these listeners after the component where it is used is unmounted. What do you think?

Reply:
Right.
Cleaning listeners when there is no `Hoverable` component in use is one of the possible upgrades. Also mouse listener can be probably added after first `touchstart`. Because it's not needed before that.

Reply:
Oh right, when there is no _other_ `Hoverable` simultaneously... So it's a bit more involved than what I originally thought. So we will do these improvements later. 
Could you still extract these two possible improvements into issues? Thanks

Reply:
Ok, issue for `Hoverable` enhancements created.